tests/test_invenio_theme.py

```diff
@@ -34,17 +34,14 @@ def test_version():
 
 def test_init(app):
     """Initialization."""
-    theme = InvenioTheme(app)
-    assert theme.menu is not None
+    InvenioTheme(app)
     assert "THEME_SITENAME" in app.config
 
 
 def test_init_app(app):
     """Initialization."""
     theme = InvenioTheme()
-    assert theme.menu is None
```

Comment:
but there exists no more `theme.menu`. so we should check against the existence of menu imported from `shared.py`?

Reply:
On second thought, I'm not against keeping the `InvenioTheme.menu` and `InvenioTheme.menu_ext` pointers... The "global" instances in `shared.py` are there so that they can be used similarly to `from invenio_db import db`. We should still have some hard-connected references though to the `InvenioTheme` extension instance.

Reply:
Should we merge `menu`  and `menu_ext` somehow? `invenio-db` uses `SQLAlchemy` from `flask-sqlalchemy` but does not introduce two global variables. Further i think that `menu_ext` is somehow useless. i don`t see the use case. i only see the use for `menu` and if we use the `register_menu` decorator also the `menu` use case is gone too.